Gemfile

```diff
@@ -16,6 +16,7 @@ group :test do
   # newer versions require ruby 2.2
   if RUBY_VERSION < "2.2.0"
     gem 'listen', '~> 3.0.0'
+    gem 'nokogiri', '1.6.8.1'
```

Comment:
It's better to relax constrains, if possible. I guess using 
```ruby
gem 'nokogiri', '< 1.7.0'
```
should be enough in this case.